rts/lra/lra-test/lra-test-arquillian-extension/src/main/java/io/narayana/lra/arquillian/ArquillianLRATckExtension.java

```diff
@@ -20,7 +20,7 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
 
-package io.narayana.lra.tck.arquillian;
+package io.narayana.lra.arquillian;
```

Comment:
Is there some specific reason why this is moved from the TCK package? I note Tck is still in the classname?

Reply:
PS if not required I favour not moving it

Reply:
yes, it's now being reused also in basic tests so not specific only to TCK.

Reply:
Should we rename the classname then? It has the word Tck in it?

Reply:
+1